### PR TITLE
Last fixes before v0.1.0 release

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/structural/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] Parameters Documentation is updated with `nf-core schema docs --format markdown --output docs/parameters.md --force`
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/images/metro_map.png
Comment (mvheetve) [inline docs/images/metro_map.png:1]: Maybe put the logo on the top right?

File: docs/images/metro_map.svg
Comment (mvheetve) [inline docs/images/metro_map.svg:1]: same comment as for .png

File: docs/images/metro_map.png
Comment (nvnieuwk) [reply docs/images/metro_map.png:1]: Good idea! Should I also put the text on the top right?

Comment (mvheetve) [inline docs/images/metro_map.png:1]: can we make it a bit more suttle? A bit smaller so there is more white space between the map and the logo? I'm splitting hairs I know...

Comment (nvnieuwk) [reply docs/images/metro_map.png:1]: Done! :p

**Review by mvheetve — APPROVED** [review]
LGTM

**Pull request merged**